src/sidecar_comms/form_cells/base.py

```diff
@@ -126,20 +134,20 @@ def update(self, data: dict) -> None:
 
 
 # --- Specific models ---
+def validate_datetime_value(value):
+    """Make sure value is a valid datetime object"""
+    if isinstance(value, str):
+        if value.endswith("Z"):
+            value = value.replace("Z", "+00:00")
+        value = datetime.fromisoformat(value)
+    return value
+
+
 class Datetime(FormCellBase):
     input_type: Literal["datetime"] = "datetime"
-    value: str = Field(default_factory=lambda: datetime.now(timezone.utc))
-
-    @validator("value", pre=True, always=True)
-    def validate_value(cls, value):
-        """Make sure value is a valid datetime string in isoformat"""
-        if isinstance(value, str):
-            if value.endswith("Z"):
-                value = value.replace("Z", "+00:00")
-            value = datetime.fromisoformat(value)
-        if isinstance(value, datetime):
-            value = value.strftime("%Y-%m-%dT%H:%M")
-        return value
+    value: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
+
+    _validate_value = validator("value", pre=True, always=True)(validate_datetime_value)
```

Comment:
Not a fan of the `fn(one_call)(another_call)` looking syntax, very non-Pythonic. Let's use the `validator` function as a decorator like in the Pydantic docs